### PR TITLE
Virtual additional info field

### PR DESCRIPTION
**Pull request by alanborning**
This change adds support for a new field "Virtual Meeting Additional Info" to the crouton metadata template. This is intended to accompany a pull request for the root server https://github.com/bmlt-enabled/bmlt-root-server/pull/309 that adds support for the field to the database. (This new version of crouton could run with the previous version of the server without any errors -- it just would always omit the new field.)

A couple of things to review please: 
- The new field is displayed using just `<div>` without a css class - is this ok, or is there an appropriate css class or should a new one be added?
- The field is shown in column 3 -- this seemed reasonable since it is data about the virtual meeting that accompanies the link shown above it. But it could go in column 2 instead ... should it? (People can customize the templates to put it either place, of course; the question is what should be the default.)

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,11 +1,11 @@
 === crouton ===
 
-Contributors: odathp, radius314, pjaudiomv, californiasteve9390, Paul N
+Contributors: odathp, radius314, pjaudiomv, californiasteve9390, Paul N, Alan B
```
Comment (radius314) [inline readme.txt:3]: This username is your wordpress account name.  Is that it?

File: croutonjs/src/js/crouton-core.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ function Crouton(config) {
...
...this}}</div>{{else}}<div class='meetsVirtually'><span class='glyphicon glyphicon-cloud'></span> {{meetsVirtually this}}</div>{{/isHybrid}}{{#if this.virtual_meeting_link}}<div><span class='glyphicon glyphicon-globe'></span> {{webLinkify this.virtual_meeting_link}}</div>{{#if this.show_qrcode}}<div class='qrcode'>{{qrCode this.virtual_meeting_link}}</div>{{/if}}{{/if}}{{#if this.phone_meeting_number}}<div><span class='glyphicon glyphicon-earphone'></span> {{phoneLinkify this.phone_meeting_number}}</div>{{#if this.virtual_meeting_additional_info}}<div>{{this.virtual_meeting_additional_info}}</div>{{/if}}{{#if this.show_qrcode}}<div class='qrcode'>{{qrCode this.phone_meeting_number}}</div>{{/if}}{{/if}}{{/isVirtual}}{{#isNotTemporarilyClosed this}}{{#unless (hasFormats 'VM' this)}}<div><a id='map-button' class='btn btn-primary btn-xs' href='https://www.google.com/maps/search/?api=1&query={{this.latitude}},{{this.longitude}}&q={{this.latitude}},{{this.longitude}}' target='_blank' rel='noopener noreferrer'><span class='glyphicon glyphicon-map-marker'></span> {{this.map_word}}</a></div><div class='geo hide'>{{this.latitude}},{{this.longitude}}</div>{{/unless}}{{/isNotTemporarilyClosed}}"
```
Comment (radius314) [inline croutonjs/src/js/crouton-core.js:57]: what happens to this if they are not upgraded to the latest root server that has this value?

Comment (alanborning) [reply croutonjs/src/js/crouton-core.js:57]: If they aren't upgraded to the latest root server that has this value, the field will just not ever be displayed (but there wouldn't be any error).

Comment (radius314) [reply croutonjs/src/js/crouton-core.js:57]: i would include something in the release notes about what version is required for this feature.

Comment (tempsaint) [reply croutonjs/src/js/crouton-core.js:57]: Why would this be in the 3rd column? Shouldn't text based additional information go in the 2nd column? 3rd column is traditionally for links (?).

Comment (pjaudiomv) [reply croutonjs/src/js/crouton-core.js:57]: Comments field shows up in first, I like that 😂

**Comment by radius314** [comment]
> * The new field is displayed using just `<div>` without a css class - is this ok, or is there an appropriate css class or should a new one be added?
> * The field is shown in column 3 -- this seemed reasonable since it is data about the virtual meeting that accompanies the link shown above it. But it could go in column 2 instead ... should it? (People can customize the templates to put it either place, of course; the question is what should be the default.)

With regard to the `<div>` I would follow whatever existing pattern exists as a first pass.  Perhaps it would be fairly simple to add a class here that is new.  Maybe someone wants the text to be red?

I would keep this in column 3 for now.  Folks have the choice to move it to the 2nd column.  When the new template features are implemented in https://github.com/bmlt-enabled/crouton/issues/256 there might be a stock profile that you include that allows for that.

File: croutonjs/src/js/crouton-core.js (hunk 2 above)
Comment (radius314) [reply croutonjs/src/js/crouton-core.js:57]: I think the question here is context of the additional information.

Comment (radius314) [reply croutonjs/src/js/crouton-core.js:57]: > Why would this be in the 3rd column? Shouldn't text based additional information go in the 2nd column? 3rd column is traditionally for links (?).

You could make an argument for either column.

**Comment by alanborning** [comment]
I took all of the suggestions and updated the pull request. In particular, I moved the new field to column 2 as @tempsaint suggested -- my thinking previously was that it would be more in context in column 3 right under a zoom URL, but if the text were for example "zoom meeting ID: 343 233 333" rather than just "meeting ID: 343 233 333" then it would work fine in column 2, since that would provide the needed context.  Also as others have noted, this would fit with the standard of having just clickable links in column 3. When we have a blog post we could note formatting the info as e.g. "zoom meeting ID: 343 233 333" as Best Practice for using the new field. I also put my wordpress id in the readme and noted that it needs version 2.15.4 of the root server.

**Comment by radius314** [comment]
> I took all of the suggestions and updated the pull request. In particular, I moved the new field to column 2 as @tempsaint suggested -- my thinking previously was that it would be more in context in column 3 right under a zoom URL, but if the text were for example "zoom meeting ID: 343 233 333" rather than just "meeting ID: 343 233 333" then it would work fine in column 2, since that would provide the needed context. Also as others have noted, this would fit with the standard of having just clickable links in column 3. When we have a blog post we could note formatting the info as e.g. "zoom meeting ID: 343 233 333" as Best Practice for using the new field. I also put my wordpress id in the readme and noted that it needs version 2.15.4 of the root server.

As i said, I don't think that you are wrong either way you do it.  It's just defining the default.

**Pull request merged**